File: 백준/Bronze/4344. 평균은 넘겠지/평균은 넘겠지.py
count = int(input())

data = []
for i in range(count):
    test = list(map(int,input().split()))
    data.append(test) 
    
total = 0
avg = 0
more_avg = 0

for i in data:
    for j in range(1,i[0]+1):
        total = total + i[j]
    avg = total / i[0]
    for k in range(1,i[0]+1):
        if i[k] > avg:
            more_avg = more_avg + 1
    if more_avg ==0:
        print('0.000%')
    else:
        print('%0.3f' % round(float(more_avg)/i[0]*100,3)+'%')
    total =0
    avg =0
    more_avg=0

File: 백준/Bronze/4344. 평균은 넘겠지/평균은 넘겠지.java
import java.util.Scanner;
import java.util.ArrayList;

public class Main{
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        int C = sc.nextInt();
        
        for(int i=0; i<C; i++){
            ArrayList<Integer> scores = new ArrayList<Integer>();
            int total =0;
            double avg =0;
            int count = sc.nextInt();
            int overAvg = 0;
            for(int j=0; j<count; j++){
                int score=sc.nextInt();
                total+=score;
                scores.add(score);
            }
            avg = total/ count;
            
            for(int s : scores){
                if(s > avg){
                    overAvg++;
                }
            }
            System.out.println(String.format("%.3f",(double)overAvg/(double)count*100)+"%");
        }
    }
}